ui/src/builder/BuilderSettings.vue

```diff
@@ -74,7 +74,7 @@ const ssbm = inject(injectionKeys.builderManager);
 const docsActive = ref(false);
 
 const component = computed(() => ss.getComponentById(ssbm.getSelectedId()));
-const readonly = computed(() => component.value.flag === "cmc");
+const readonly = computed(() => component.value.isCodeManaged);
```

Comment:
question: is computed here still required? 
Can't we use this `v-if="component.isCodeManaged"`

Reply:
I left it because it helps with readability, but probably overkill.